### PR TITLE
Improve wording around forenames parameter

### PR DESCRIPTION
**Pull request by Krenair**

### CODE REVIEW

File: source/api-reference.html.md.erb [hunk 1]
```diff
@@ -130,7 +130,7 @@ The number of the passport being checked, which is an integer between 1 and 8999
 String of maximum 30 characters containing the surname as on the passport.
 
 #### forenames
-String of maximum 30 characters, including spaces between forenames. The forenames must be the same as on the passport.
+List of strings of maximum 30 characters, including spaces between forenames. The forenames must be the same as on the passport.
```
Comment (Krenair) [inline source/api-reference.html.md.erb]:
```suggestion
List of strings. When concatenated together (including spaces between items) totalling of maximum 30 characters. The forenames must be the same as on the passport.
```

Comment (bravegrape) [reply source/api-reference.html.md.erb]: That's remarkably close to what @bethwalton and I came up with: 

List of strings containing the forenames on the passport. The concatenated strings, including spaces between forenames, must be maximum 30 characters.

Comment (Krenair) [reply source/api-reference.html.md.erb]: I prefer your one.

Comment (Krenair) [reply source/api-reference.html.md.erb]:
```suggestion
List of strings containing the forenames on the passport. The concatenated strings, including spaces between forenames, must be maximum 30 characters.
```

Comment (bravegrape) [reply source/api-reference.html.md.erb]: If you commit that version then, I can approve an away we go! 🚀

**Pull request merged**